code/modules/admin/investigation.dm

```diff
@@ -0,0 +1,34 @@
+#define INVESTIGATION_SUBJECT_MAIN "main"
+#define INVESTIGATION_SUBJECT_HREF "href"
+#define INVESTIGATION_SUBJECT_QDEL "qdel"
+
+/client/proc/investigation_panel(subject in list(INVESTIGATION_SUBJECT_MAIN, INVESTIGATION_SUBJECT_HREF, INVESTIGATION_SUBJECT_QDEL))
```

Comment:
This misses the point of this system, which is to filter the logs to something useful for actual admin work. Admins don't care about qdel logs, and it's not even clear they should access the full logs.